installer/install.sh

```diff
@@ -266,27 +266,28 @@ function read_cluster_ip() {
 
 # To execute ansible task
 function run_ansible_tasks() {
-  echo "Performing clean and running the installer. You will be asked for your password."
+  echo -e "Performing clean and running the installer ..."
+  echo -e "May you will be asked for your System Admin Password(root/sudo)."
 
   cd ${SCRIPT_ABSOLUTE_PATH}
   cd .. && make clean &>/dev/null
 
   set -e
   echo "Installing roles to ${SCRIPT_ABSOLUTE_PATH}/roles"
-  ansible-galaxy install -r ./installer/requirements.yml --roles-path="${SCRIPT_ABSOLUTE_PATH}/roles" --force
+  sudo ansible-galaxy install -r ./installer/requirements.yml --roles-path="${SCRIPT_ABSOLUTE_PATH}/roles" --force
```

Comment:
Not sure this is a good idea. If possible I would like to remove any need for Sudo or root perms. The less we need to escalate the better